static/src/javascripts/es6/projects/common/modules/crosswords/hidden-input.js

```diff
@@ -20,7 +21,7 @@ export default class HiddenInput extends React.Component {
         if (detect.isBreakpoint({ max: 'mobile' })) {
             fastdom.read(function () {
                 const offsets = bonzo(React.findDOMNode(this.refs.input)).offset();
-                scroller.scrollTo(offsets.top - (offsets.height * 1.5), 250, 'easeOutQuad');
+                scroller.scrollTo(offsets.top - (offsets.height * 1.5) - $('.crossword__sticky-clue').offset().height, 250, 'easeOutQuad');
```

Comment:
:unamused: I hate that the sticky clue leaks into the input field component. Any better ideas?